lib/api/client-commands/perform.js

```diff
@@ -81,7 +81,10 @@ class Perform extends EventEmitter {
 
       if (cbResult instanceof Promise) {
         clearTimeout(this.timeoutId);
-        cbResult.then(result => {
+        cbResult.then(async result => {
+          if (result instanceof Actions) {
```

Comment:
what about the other perform() call above?

Reply:
In that case it won't end up here as the result in that case would be resolve value of `actions.click` which will not be an instance of Actions API